src/app/shared/button/button.component.ts

```diff
@@ -0,0 +1,18 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {Constants} from '../constants';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent {
+
+  @Input() buttonStyle: string;
+  @Input() isBig: boolean;
+  @Output() buttonClicked: EventEmitter<any> = new EventEmitter<any>();
+
+  secondStyle = Constants.Button.Secondary;
+  tertiaryStyle = Constants.Button.Tertiary;
+
```

Comment:
Is this empty line required?

Reply:
No, in theory you can write all of the code in one line, but extra lines makes it more readable, don't they?

Reply:
Fair enough.